zombies.js

```diff
@@ -208,6 +205,106 @@
  * @return {string/boolean}   Weapon name or false if nothing is equipped.
  */
 
+class Player {
+  constructor(name, health, strength, speed) {
+    this.name = name;
+    this.health = health;
+    this.strength = strength;
+    this.speed = speed;
+    this._pack = [];
+    this._maxHealth = this.health;
+    this.isAlive = true;
+    this.equipped = false;
+  }
+
+  getPack() {
+    return this._pack;
+  }
+
+  getMaxHealth() {
```

Comment:
getMaxHealth should be in the constructor and placed on this.

It should only return the this._maxHealth value.